pages/form/ConfirmWFH.jsx

```diff
@@ -0,0 +1,139 @@
+import { 
+  Box,
+  Heading,
+  Flex,
+  Text
+} from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+import useForm from "../../components/FormProvider";
+import Q1Progress from "../../public/images/progress-bar/q1-progress-dots.svg";
+import Q3Cloud from "../../public/images/clouds/cloud-q3.svg";
+import LinkButton, { BackButton } from "../../components/LinkButton/LinkButton";
+import { useRouter } from "next/router";
+import { sendLogs } from "../../utils/sendLogs";
+
+import ConfirmDetailsContainer from "../../components/ConfirmDetailsContainer/ConfirmDetailsContainer";
+
+/* for now, we pass hard-coded data from the porent component (icon and title), 
+eventually will render based on what user selects during previous steps:
+*/
+import House from "../../public/images/other/House.svg";
```

Comment:
just a small typo here (was causing a build error with Vercel deployment)
 
```suggestion
import House from "../../public/images/other/house.svg";
```

Reply:
good pickup @hqtan , I changed casing for all icons I added and associated paths 👍